Makefile

```diff
@@ -121,6 +121,15 @@ lint-go-code: $(GOLANGCI_LINT_BIN)
 $(GOLANGCI_LINT_BIN):
 	$(Q)curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b ./out v1.17.1
 
+.PHONY: courier
+## Validate manifests using operator-courier
+courier:
+	$(Q)python3 -m venv ./out/venv3
```

Comment:
I like the approach of creating a virtual environment for installing operator-courier. Can we do a similar thing for `yamllint` inside `lint-yaml` target. Then we can remove `yamllint` from the container image.
Refere: https://github.com/redhat-developer/service-binding-operator/blob/e21d5b1f2e9b874c9172296699ed4cb54a3d572c/openshift-ci/Dockerfile.tools#L20

Reply:
This change need not be in this PR.

Reply:
Thank you, I'll create another PR for this.